README.md

```diff
@@ -39,6 +39,11 @@ If you wish you can run this command under a virtual environment.
    when getting images, Avocado will create a snapshot to avoid conflicts. Also
    domains created by tests have unique names to avoid collisions.
 
+ * Creating domains easily: You can use the `.get_generic_domain()` method to
+   create a generic domain, based on a template (Jinja2).
```

Comment:
IMO, if are going to interact often with such functionality, it's important that they can keep in their minds, that is, it's be nice if this was called something like `get_domain()` instead of `get_generic_domain()`.  Special, that is, non generic domains, can be achieved by either special arguments or special methods.

Reply:
Yes, we have a `get_generic_image()` and a `get_domain()`, my bad.